dex/src/main/scala/com/wavesplatform/dex/cli/WavesDexCli.scala

```diff
@@ -173,12 +173,15 @@ object WavesDexCli extends ScoptImplicits {
 
   // noinspection ScalaStyle
   def makeSnapshots(args: Args): Unit = {
+    lazy val (_, matcherSettings) = loadAllConfigsUnsafe(args.configPath)
+    val updatedArgs = matcherSettings.cli.defaultArgs.coverEmptyValues(args)
```

Comment:
We do this for all arguments, see below `OParser.parse(parser, rawArgs, Args())`